cmd/podman/generate/kube.go

```diff
@@ -48,6 +56,22 @@ func init() {
 	flags.StringVarP(&kubeFile, filenameFlagName, "f", "", "Write output to the specified path")
 	_ = kubeCmd.RegisterFlagCompletionFunc(filenameFlagName, completion.AutocompleteDefault)
 
+	clusterFlagName := "cluster"
+	flags.BoolVarP(&kubeOptions.Cluster, clusterFlagName, "c", false, "Deploy the generated yaml to a Kubernetes cluster")
+	_ = kubeCmd.RegisterFlagCompletionFunc(clusterFlagName, completion.AutocompleteDefault)
+
+	kubeconfigFlagName := "kubeconfig"
+	flags.StringVarP(&kubeOptions.Kubeconfig, kubeconfigFlagName, "k", os.Getenv("KUBECONFIG"), "Path to the kubeconfig file for the Kubernetes cluster")
+	_ = kubeCmd.RegisterFlagCompletionFunc(kubeconfigFlagName, completion.AutocompleteDefault)
+
+	namespaceFlagName := "ns"
+	flags.StringVarP(&kubeOptions.Namespace, namespaceFlagName, "", "", "The namespace to deploy the workload to on the Kubernetes cluster")
+	_ = kubeCmd.RegisterFlagCompletionFunc(namespaceFlagName, completion.AutocompleteDefault)
+
+	caCertFileFlagName := "ca-cert-file"
+	flags.StringVarP(&kubeOptions.CACertFile, caCertFileFlagName, "", "", "Path to the CA cert file for the Kubernetes cluster.")
```

Comment:
Can we make setting any of these without setting `cluster` first an error?

Reply:
Yup, fixed.